conf/broker.conf

```diff
@@ -340,6 +340,22 @@ tokenSecretKey=
 # tokenPublicKey=file:///my/public.key
 tokenPublicKey=
 
+### --- SASL Authentication Provider --- ###
+
+# Whether Use SASL Authentication or not.
+# TODO: used to bypass web resource check. will remove it after implementation the support.
```

Comment:
TODO ?

Reply:
Thanks @eolivelli , yes, there is a planed work. 
current step is for client <-> broker. will add the web-resource/http support in the TODO issue.

Reply:
by this parameter we could bypass the web-resource/http checking temporary.